src/js/components/Animate.js

```diff
@@ -3,7 +3,7 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { findDOMNode } from 'react-dom';
-import TransitionGroup from 'react-addons-transition-group';
+import TransitionGroup from 'react-transition-group/TransitionGroup';
```

Comment:
can we do ES6 import instead?

`import { TransitionGroup } from 'react-transition-group';`